src/components/companies/list/SectionedCompanyList.tsx

```diff
@@ -1,126 +1,20 @@
-import {
-  Accordion,
-  AccordionContent,
-  AccordionItem,
-  AccordionTrigger,
-} from "@/components/ui/accordion";
 import { CompanyCard } from "./CompanyCard";
 import type { RankedCompany } from "@/types/company";
-import {
-  SECTOR_ORDER,
-  useSectorNames,
-} from "@/hooks/companies/useCompanyFilters";
-import { useTranslation } from "react-i18next";
 
 interface SectionedCompanyListProps {
   companies: Omit<RankedCompany, "rankings" | "goals" | "initiatives">[];
-  sortBy: string;
 }
 
-export function SectionedCompanyList({
-  companies,
-  sortBy,
-}: SectionedCompanyListProps) {
-  const { t } = useTranslation();
-  const sectorNames = useSectorNames();
-
-  // Group companies by sector
-  const companiesBySector = companies.reduce(
-    (acc, company) => {
-      const sectorCode =
-        company.industry?.industryGics?.sectorCode || "unknown";
-      if (!acc[sectorCode]) {
-        acc[sectorCode] = [];
-      }
-      acc[sectorCode].push(company);
-      return acc;
-    },
-    {} as Record<
-      string,
-      Omit<RankedCompany, "rankings" | "goals" | "initiatives">[]
-    >,
-  );
-
-  // Sort sectors by predefined order
-  const sortedSectors = Object.keys(companiesBySector).sort((a, b) => {
-    const indexA = SECTOR_ORDER.indexOf(a as (typeof SECTOR_ORDER)[number]);
-    const indexB = SECTOR_ORDER.indexOf(b as (typeof SECTOR_ORDER)[number]);
-    if (indexA === -1) return 1;
-    if (indexB === -1) return -1;
-    return indexA - indexB;
-  });
-
-  // Sort companies within each sector based on the selected sort option
-  Object.keys(companiesBySector).forEach((sector) => {
-    companiesBySector[sector].sort((a, b) => {
-      switch (sortBy) {
-        case "emissions_reduction":
-          return b.metrics.emissionsReduction - a.metrics.emissionsReduction;
-        case "total_emissions":
-          return (
-            (b.reportingPeriods[0]?.emissions?.calculatedTotalEmissions || 0) -
-            (a.reportingPeriods[0]?.emissions?.calculatedTotalEmissions || 0)
-          );
-        case "scope3_coverage":
-          return (
-            (b.reportingPeriods[0]?.emissions?.scope3?.categories?.length ||
-              0) -
-            (a.reportingPeriods[0]?.emissions?.scope3?.categories?.length || 0)
-          );
-        case "name_asc":
-          return a.name.localeCompare(b.name);
-        case "name_desc":
-          return b.name.localeCompare(a.name);
-        default:
-          return 0;
-      }
-    });
-  });
-
+export function SectionedCompanyList({ companies }: SectionedCompanyListProps) {
```

Comment:
This should probably be renamed to `CompanyList` now as it is no longer sectioned?